UnleashClient/features/Feature.py

```diff
@@ -118,6 +118,8 @@ def get_variant(self, context: dict = None, skip_stats: bool = False) -> dict:
                 LOGGER.warning("Error selecting variant: %s", variant_exception)
         if not skip_stats:
             self._count_variant(cast(str, variant["name"]))
+
+        variant["is_feature_enabled"] = is_feature_enabled
         return variant
```

Comment:
If we take the above, this is just to make it easier to accept the suggestion without another PR
```suggestion
```